test/test_data/libxkbcommon.py

```diff
@@ -0,0 +1,21 @@
+mapping_test_data = [
+    {
+        "product": "libxkbcommon",
+        "version": "0.8.1",
+        "version_strings": ["libxkbcommon 0.8.1"],
+    }
+]
+package_test_data = [
+    {
+        "url": "http://mirror.centos.org/centos/7/os/x86_64/Packages/",
+        "package_name": "libxkbcommon-0.7.1-3.el7.x86_64.rpm",
+        "product": "libxkbcommon",
+        "version": "0.7.1",
+    },
+    {
+        "url": "https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/30/Everything/x86_64/os/Packages/l/",
+        "package_name": "libxkbcommon-0.8.3-1.fc30.x86_64.rpm",
+        "product": "libxkbcommon",
+        "version": "0.8.3",
+    },
```

Comment:
` },` I think that this `,` shouldn't be at the end, but since all other similar checkers had this, so I added this too.

Reply:
Both the trailing comma and the lack of trailing comma are allowed in python.  I prefer the trailing commas here because then when a new contributor tries to add a test to the end, they don't have to also remember to add a comma to the previous one.  Why make life more annoying than it has to be by having one entry be slightly different in a way that's hard to see at a glance?